src/js/modules/products.js

```diff
@@ -155,8 +155,27 @@ const productWaitlistSubmitted = $('.product-waitlist__submitted');
 if (productWaitlistForm.length > 0 && productWaitlistSubmitted.length > 0) {
 	$('.product-waitlist__form form').on('submit', function (e) {
 		e.preventDefault();
-		console.log('click');
 		productWaitlistForm.addClass('d-none');
 		productWaitlistSubmitted.removeClass('d-none');
 	});
 }
+
+// popover
+if ($('[data-toggle="popover"]').length) {
+	$(function () {
+		$('[data-toggle="popover"]').popover({
+			delay: {
+				show: 100,
+			},
+		});
+	});
+}
+
+// Dismissable popover click out side
+$('body').on('click', function (e) {
```

Comment:
this function should be run only if the popover element is in the page and open

Reply:
sure okay

Reply:
Updated